### PR TITLE
Added missing FRLG breeding rng method

### PR DESCRIPTION
**Pull request by Real96**
Code taken from [here](https://github.com/Lincoln-LM/Finder-ToolBox/commit/bf75fd531369af8a8bacaaebfc0a2d8fcde4d7eb) and [here](https://github.com/Lincoln-LM/Finder-ToolBox/commit/c129e822dde485d389db5a2cf6b1d1bf60cce6c2).

Thanks to [Lincoln-LM](https://github.com/Lincoln-LM/)!

### CODE REVIEW

File: Source/Forms/Gen3/Eggs3.ui [hunk 1]
```diff
@@ -103,6 +103,9 @@
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabEggSelection">
+     <property name="currentIndex">
+      <number>2</number>
+     </property>
```
Comment (Admiral-Fish) [inline Source/Forms/Gen3/Eggs3.ui:108]: Please remove this

**Review by Admiral-Fish — CHANGES_REQUESTED** [review]
Does RS require this as well?

File: Source/Forms/Gen3/Eggs3.ui (hunk 1 above)
Comment (Real96) [reply Source/Forms/Gen3/Eggs3.ui]: Done

**Comment by Real96** [comment]
> Does RS require this as well?

No one hit this method on RS, so i assume it isn't needed. If we get this in future i'll add it in case

**Pull request merged**